.pipelines/templates/release-checkout-pwsh-repo.yml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - pwsh: |
+      Write-Verbose -Verbose "Deploy Box Product Pathway Does Not Support the `"checkout`" task"
+      git clone --depth 1 https://$(mscodehubCodeReadPat)@mscodehub.visualstudio.com/PowerShellCore/_git/PowerShell '$(Pipeline.Workspace)/PowerShell'
```

Comment:
depth should be a parameter

Reply:
If we make a separate stage for CL and tools upload, then I'm seeing this template as unnecessary...